embc-app/ClientApp/src/app/volunteer/pages/referral-maker/referral-maker.component.html

```diff
@@ -15,170 +15,163 @@ <h1>Identify Supports</h1>
       </div>
       <div class="feature-block card d-block border-0">
         <div>Incident start date &amp; time: {{registration?.incidentTask?.startDate | dateTimeFormatPipe}}</div>
-        <h3 class="mt-3">
-          Support start date &amp; time: {{registration?.incidentTask?.startDate | dateTimeFormatPipe}}
-        </h3>
+        <div class="row">
+          <div class="col">
+            <span class="align-bottom">
+              <span class="h3">Support start date &amp; time: {{defaultDate | dateTimeFormatPipe}} <span *ngIf="!defaultDate">Please enter a valid date.</span></span>
```

Comment:
This "error message" is fine, but if the user closes the override panel then the date should revert to the incident task date (but it doesn't).

Reply:
I have fixed this in a new commit.